src/Host.cpp

```diff
@@ -1186,3 +1192,31 @@ QString Host::readProfileData(const QString& item)
 
     return ret;
 }
+
+// makes fonts in a given package/module be available for Mudlet scripting
+// does not install font system-wide
+void Host::installPackageFonts(const QString &packageName)
+{
+    auto packagePath = mudlet::getMudletPath(mudlet::profilePackagePath, getName(), packageName);
+
+    QDirIterator it(packagePath, QDirIterator::Subdirectories);
+    while (it.hasNext()) {
+        auto filePath = it.next();
+
+        if (filePath.endsWith(QLatin1String(".otf"), Qt::CaseInsensitive) || filePath.endsWith(QLatin1String(".ttf"), Qt::CaseInsensitive) ||
+            filePath.endsWith(QLatin1String(".ttc"), Qt::CaseInsensitive) || filePath.endsWith(QLatin1String(".otc"), Qt::CaseInsensitive)) {
+            if (QFontDatabase::addApplicationFont(filePath) == -1) {
```

Comment:
We could log the id number for future reference (for accessing `[static] QStringList QFontDatabase::applicationFontFamilies(int id)` and for deletion although that is not contemplated at this time). Also we could log the file associated with the font being loaded so that we can avoid reloading it again).  Note that it is not possible to get the id after the `addApplicationFont` has completed and there is no method to iterate directly through the `QFontDatabase` to examine the existing entries - so storing the details here may be useful.

Reply:
It succeeds and returns a different ID. I don't believe we need to log the id's however, there is no usecase for removing fonts and I'm keeping it on the simple side to avoid overengineering.

Reply:
I would content that we should not be loading the *same* font files repeatedly without knowing how Qt handles it.  I still think we should at least note the absolute file-name used (or perhaps just the base file name) against the Id that Qt allocates to it in, say a `QMap<int, QString>` or for better lookup performance as a `QMap<QString, int>` and then avoid loading the same name if it is already in that `QMap` and thus in the application's `QFontDatabase`.  After all, it is pointless and wastes time if multi-playing if different profiles load the same font from a common location.

Reply:
OK, that is a fair point. I've made the adjustments requested!